management/src/main/resources/application.yml

```diff
@@ -3,5 +3,24 @@ spring:
     view:
       prefix: /WEB-INF/jsp/
       suffix: .jsp
+  rabbitmq:
+      host: localhost
+      port: 5672
+      username: ${RABBIT_USERNAME}
+      password: ${RABBIT_PASSWORD}
+  cloud:
+    stream:
+      bindings:
+        createProductsChannel:
+          destination: productsExchange
+          contentType: application/json
+          group: productsQueue
```

Comment:
what is group for?

Reply:
The group is the queue

Reply:
No it's not, please research before giving an wrong answer, this could lead a team confusion and even worst, making your team peer learning something is not true. 

@iuresti The groups defines the consumer group the service belongs: 

> Spring Cloud Stream models this behavior through the concept of a consumer group. (Spring Cloud Stream consumer groups are similar to and inspired by Kafka consumer groups.) Each consumer binding can use the spring.cloud.stream.bindings.<channelName>.group property to specify a group name. For the consumers shown in the following figure, this property would be set as spring.cloud.stream.bindings.<channelName>.group=hdfsWrite or spring.cloud.stream.bindings.<channelName>.group=average.

[Reference Link](https://docs.spring.io/spring-cloud-stream/docs/current/reference/htmlsingle/)

Reply:
Suena bien para kafka, pero no será correcto lo que menciona Juan para RabbitMQ ?

Reply:
Ok i will research more about that thanks, but i based my comment in this page http://www.jorgehernandezramirez.com/2017/05/10/spring-cloud-stream-rabbitmq/
also in the rabbit client i can see that the group defined is also a queue
![screenshot from 2018-08-08 12-55-20](https://user-images.githubusercontent.com/38140857/43855280-b5d532da-9b0a-11e8-8b3c-8024ab9992e3.png)